packages/magento-product/components/ProductPageMeta/ProductPageMeta.graphql

```diff
@@ -1,7 +1,26 @@
 fragment ProductPageMeta on ProductInterface {
   ...ProductLink
+  sku
   name
   url_key
   meta_title
   meta_description
+  price_range {
+    maximum_price {
+      final_price {
+        value
+        currency
+      }
+      regular_price {
+        value
+      }
+    }
```

Comment:
Is there a reason we're using maximum_price here? In the case of a configurable with multiple prices this will return the higher price. Is that by design?

Reply:
I thought it would be safer to return the highest value here as min price could sometimes be a part of the product given bundles for example but in the end, it comes down to the kind of products a shop sells. Now I think the minimum price would be a more universal solution as many systems advertise the price in the "starting from" format. 
Done

Reply:
I think that is a good solution :)